src/selection_criteria.rs

```diff
@@ -216,9 +250,34 @@ impl ReadPreference {
             doc.insert("tags", tags);
         }
 
+        if let Some(hedge) = hedge {
+            doc.insert("hedge", doc! { "enabled": hedge.enabled });
+        }
+
         doc
     }
 }
 
 /// A read preference tag set. See the documentation [here](https://docs.mongodb.com/manual/tutorial/configure-replica-set-tag-sets/) for more details.
 pub type TagSet = HashMap<String, String>;
+
+#[cfg(test)]
+mod test {
```

Comment:
No new spec tests were added as part of the work for the hedged read option, so I just made a minimal test to ensure that the hedge document is correctly added when serializing the ReadPreference. I didn't think it was necessary to add any integration tests (e.g. with command monitoring) due to the fact that `ReadPreference::into_document` is really the only part of this module that's used for the other components that would be tested by that.